.travis.yml

```diff
@@ -50,14 +50,14 @@ before_script:
   - composer global require phpunit/phpunit:@stable fabpot/php-cs-fixer --no-update
   - composer global update --prefer-dist --no-interaction
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
-  - if [ "$ACL_VERSION" != "" ]; then composer require "symfony/security-acl:${ACL_VERSION}" --no-update; fi;
   - travis_wait composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
   - export PATH=$HOME/.local/bin:$PATH
   - pip install -r Resources/doc/requirements.txt --user `whoami`
 
 script:
  - if [ "$CS_FIXER" = "run" ]; then make cs_dry_run ; fi;
- - make test
+ - if [ "$DOCS" = "run" ]; then make docs ; fi;
+ - if [ "$SKIP_TESTS" != true ]; then make test ; fi;
```

Comment:
Maybe this could be replaced with `make $TARGET`, and have target be one of `test`, docs`,or `cs_dry_run`